src/stream/src/executor/top_n/group_top_n_appendonly.rs

```diff
@@ -103,13 +101,18 @@ impl<K: HashKey, S: StateStore, const WITH_TIES: bool>
         group_by: Vec<usize>,
         state_table: StateTable<S>,
         watermark_epoch: AtomicU64Ref,
-        ctx: ActorContextRef,
+        ctx: &ActorContext,
     ) -> StreamResult<Self> {
         let metrics_info = MetricsInfo::new(
             ctx.streaming_metrics.clone(),
             state_table.table_id(),
             ctx.id,
-            "GroupTopN",
+            "AppendOnlyGroupTopN",
```

Comment:
Previously it was `GroupTopN`. Was it a bug?

Reply:
Perhaps not, here is inside `InnerAppendOnlyGroupTopNExecutor`